brackend/db/models.py

```diff
@@ -44,11 +44,23 @@ class Tournament(Base):
     users = association_proxy("user_tournaments", "user")
     brackets = relationship("Bracket", backref="tournaments")
 
+    def __init__(self, data):
+        super.__init__(self, data)
+        self.owner = None
+
     def __repr__(self):
         return f"Tournament(id={self.id}, name={self.name})"
 
+    def add_owner_info(self, data):
```

Comment:
Do you think there's a way to do this with a orm relationship instead of like this? I guess this is fine for now.

Reply:
There definitely is... but I couldn't figure out how to do a relationship via a join table with some condition